app/search.py

```diff
@@ -8,19 +8,52 @@
 import webbrowser
 from googlesearch import search
 import sys
+import os
 
 def open_browser(urls):
     for url in urls:
         webbrowser.open_new_tab(url)
 
-def search_my_term(term, count):
-    return [url for url in search(term, stop=count)]
+def search_my_term(term,s, count):
```

Comment:
never use variables like s, x, y z etc., always use a fully qualified name, in this case `start`, also make it an optional parameter. By default 0?